packages/components/src/Stack/Stack.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { system } from '@marigold/system';
+import { Box } from '../Box';
+import flattenChildren from 'react-flatten-children';
+import { Theme } from '@marigold/system';
+
+type StackProps = {
+  space?: Theme['space'];
+  align?: 'left' | 'right' | 'center';
+};
+
+export const Stack = system<StackProps, 'div'>(
+  ({ space = 0, align = 'left', className, children, ...props }) => {
+    let stackItems = flattenChildren(children);
+    let spaceValue: number = +space;
```

Comment:
Already a Number. Don't copy/paste code blindly :P

Reply:
But then I get an error from p and pt.
![Bildschirmfoto 2021-02-17 um 08 50 05](https://user-images.githubusercontent.com/59875255/108172446-28e40100-70fd-11eb-985e-ab46321746a3.png)

Reply:
Yeah, because spaces can also be an object